python/lsst/daf/butler/registry/queries/expressions/parser/exprTree.py

```diff
@@ -335,6 +361,12 @@ class IsIn(Node):
     """
 
     def __init__(self, lhs: Node, values: list[Node], not_in: bool = False):
+        # All values must be literals or binds (and we allow simple identifiers
```

Comment:
Does this need to delegate to `super()`?  I see that another derived class did.

Reply:
It also calls `Node.__init__`, I need to change all subclasses to use `super()`.

Reply:
You probably looked at another class - `RangeLiteral` was indeed missing call to base class `__init__`, I fixed that.